ui/app/AppLayouts/Onboarding2/pages/LoginPage.qml

```diff
@@ -0,0 +1,290 @@
+            width: Math.min(380, root.availableWidth)
+            spacing: Theme.bigPadding
+
+            StatusBaseText {
+                Layout.fillWidth: true
+                text: root.title
+                font.pixelSize: 22
+                font.bold: true
+                wrapMode: Text.WordWrap
+                horizontalAlignment: Text.AlignHCenter
+            }
+            StatusBaseText {
+                Layout.fillWidth: true
+                Layout.topMargin: -Theme.padding
+                text: qsTr("How would you like to log in to Status?")
+                color: Theme.palette.baseColor1
+                wrapMode: Text.WordWrap
+                horizontalAlignment: Text.AlignHCenter
+            }
+
+            OnboardingFrame {
+                Layout.fillWidth: true
+                contentItem: ColumnLayout {
+                    spacing: 20
+                    StatusImage {
+                        Layout.alignment: Qt.AlignHCenter
+                        Layout.preferredWidth: Math.min(250, parent.width)
+                        Layout.preferredHeight: Math.min(250, height)
+                        source: Theme.png("onboarding/status_login_seedphrase")
+                        mipmap: true
+                    }
+                    StatusBaseText {
+                        Layout.fillWidth: true
+                        text: qsTr("Log in with recovery phrase")
+                        font.pixelSize: Theme.secondaryAdditionalTextSize
+                        font.bold: true
+                        wrapMode: Text.WordWrap
+                        horizontalAlignment: Text.AlignHCenter
+                    }
+                    StatusBaseText {
+                        Layout.fillWidth: true
+                        Layout.topMargin: -Theme.padding
+                        text: qsTr("If you have your Status recovery phrase")
+                        font.pixelSize: Theme.additionalTextSize
+                        wrapMode: Text.WordWrap
+                        horizontalAlignment: Text.AlignHCenter
+                        color: Theme.palette.baseColor1
+                    }
+                    StatusButton {
+                        objectName: "btnWithSeedphrase"
+                        Layout.fillWidth: true
+                        text: qsTr("Enter recovery phrase")
+                        font.pixelSize: Theme.additionalTextSize
+                        onClicked: root.loginWithSeedphraseRequested()
+                    }
+                }
+            }
+
+            OnboardingButtonFrame {
+                Layout.fillWidth: true
+                id: buttonFrame
+                contentItem: ColumnLayout {
+                    spacing: 0
+                    ListItemButton {
+                        objectName: "btnBySyncing"
+                        Layout.fillWidth: true
+                        text: qsTr("Log in by syncing") // FIXME wording, "Log in by pairing"?
```

Comment:
Did we confirm if we want Pairing instead? cc @benjthayer 
I personally like it better